src/containers/HelpDropdownContainer.ts

```diff
@@ -0,0 +1,18 @@
+import { KialiAppState } from '../store/Store';
+import { connect } from 'react-redux';
+import HelpDropdown from '../components/Nav/HelpDropdown';
+
+import { HelpDropdownActions } from '../actions/HelpDropdownActions';
+
+const mapStateToProps = (state: KialiAppState) => ({
+  status: state.statusState.status,
+  products: state.statusState.products,
+  warningMessages: state.statusState.warningMessages
+});
+
+const mapDispatchToProps = (dispatch: any) => ({
+  refresh: () => dispatch(HelpDropdownActions.refresh())
+});
+
+const HelpDropdownConnected = connect(mapStateToProps, mapDispatchToProps)(HelpDropdown);
```

Comment:
This comment is due I'm learning redux, too.
So HelpDropdownConnected is the HelpDropdown component but linked with the reducers, which will bind the data from the store with the component.

Reply:
Yes with this you use the component HelpDropdown an connect to him the properties and state  of redux.
In line 7 I am passing to this components status, products and warningMessages like a properties, I asigned them the actual value of the state in redux. In the line 13 I am passing actions to the component like props, in this case I am passing the method refresh of HelpDropdownActions defined in actions/HelpDropdownActions, so in the component I can use this method with this.props.refresh